### ISSUE TITLE
YouTube DASH: Playback from saved position refuses to continue

### ISSUE THREAD

**Issue by notargets** [xylosper/bomi#360]
In bomi 0.9.10, regardless of what format I choose in the Network section of the preferences, a DASH format is selected in case of all YouTube videos. The problem with this is that unless the saved position is near the beginning of the video, playback continues only after 2-3 minutes have passed—in the meanwhile, the video area turns black.

When I select a format other than DASH from the context menu while waiting, bomi becomes unresponsive for about 30 seconds and then plays the video. If I stop and play it again at this point, it continues after a few seconds. If I choose another video or restart bomi, the whole thing starts again since the format reverts back to DASH.

Another issue that I've just noticed is that mp4 is selected in the context menu even if webm is selected in the preferences.

**Comment by xylosper** [comment]
> In bomi 0.9.10, regardless of what format I choose in the Network section of the preferences, a DASH format is selected in case of all YouTube videos (according to the context menu).

As you can see from Play > Stream Format menu, YouTube does not provide FPS information for non-DASH format which results to select DASH preferentially. Maybe I have to add such an choice of 'unknown' or 'ignore' in FPS combo box.

> Another issue that I've just noticed is that mp4 is selected in the context menu even if webm is selected in the preferences.

Usually, you cannot find exactly matching format, and bomi tries to match format in order of res. -> fps -> container. If you have smarter idea to select format heuristically, please let me know.
